Changelog.md

```diff
@@ -1,2 +1,2 @@
 # Changelog
-- Add Harmony Patch to stop Universum vacuum terrain from being polluted
```

Comment:
This should not be removed. Only clear the Changelog if it is leftovers from the last update